client/src/Components/ShareDialog/style.js

```diff
@@ -0,0 +1,23 @@
+import { makeStyles } from '@material-ui/core';
+const useStyles = makeStyles((theme) => ({
+	btn: {
+		color: 'white',
+		backgroundColor: '#4CB7A7',
+		borderRadius: 61,
+		width: 272,
+		height: 68,
+		fontSize: 25,
+		[theme.breakpoints.down('425')]: {
+			width: 200,
+			height: 60,
+			fontSize: 20,
+		},
+		[theme.breakpoints.down('382')]: {
+			width: 150,
+			height: 50,
+			fontSize: 15,
+		},
```

Comment:
Just wanted to share that some shops adhere to a "mobile first" convention with breakpoint styles, where the base styles are for mobile and then larger breakpoints override with `min-width` or `breakpoint.up`. But this is totally legitimate too!

Reply:
yeah agree but we had to start with the website first cause we don't have time